spec/spec.md

```diff
@@ -565,7 +568,7 @@ Embedded or referenced test vectors.
 
 ### Normative References
 
-[[def: OIDC]]
+[[def: OIDC,Open ID Connect (OIDC)]]
```

Comment:
can we just do [[def:Open ID Connect (OIDC)]]?

Reply:
same here, as above. will display as OIDC in the references but then I wanted to refer/link as [Open ID Connect (OIDC) for the first time it is mentioned